Gemfile

```diff
@@ -44,6 +44,7 @@ gem 'devise'
 gem 'pundit'
 gem 'rqrcode'
 gem 'request_store'
+gem 'json-jwt'
```

Comment:
Given JWTs are complicated and full of foot cannons, have you checked to see which ruby jwt libraries provide the most comprehensive support/have an active developer?

Reply:
The repo for the this gem is here... https://github.com/nov/json-jwt

Reply:
I think I'd prefer using a library that doesn't have an individual contributor as a maintainer